### DISCUSSION TITLE
Kerning between "Y" and "o" is huge.

### DISCUSSION THREAD

**Discussion by calliesniffs** [ivodolenc/aspekta#24] [Ideas]
### Steps to Reproduce

This is a great typeface, but there's an issue whenever you type any word with a capital "Y" followed by an "o". It seems like the kerning is off or there isn't a ligature in place to consider the letterform, so it always stands out and looks very strange.
<img width="569" alt="image" src="https://github.com/ivodolenc/aspekta/assets/171296206/d06d8f11-ba2b-4cd3-bfda-4ca050e1623f">

### Current Behavior

Kerning is off, image is attached in the below section.

### Expected Behavior

Would expect it to look more like the below on this image:
<img width="958" alt="image" src="https://github.com/ivodolenc/aspekta/assets/171296206/a6c6230b-c899-47f1-803c-1a7c92a6ca7e">

### Version

1.0.0

**Comment by ivodolenc** [comment]
Hi, yes I agree, kernings should be improved for some other letters as well.

Since this is a time consuming process, I'm open for PRs, if anyone is interested in helping out.

Let's leave this open for reminder. I will come back to this as soon as I have more time.

**Comment by xandermarsh** [comment]
Resurfacing this one since it's the singular issue that is causing me pause on this family
